desktop/src/reports/dialogs/column-picker-dialog/column-picker-dialog.component.scss

```
@@ -0,0 +1,200 @@
+    border: 1.5px solid #e2e8f0;
+    color: #64748b;
+    font-family: inherit;
+
+    &--selected {
+      background: rgba(39, 177, 255, 0.1);
+      border-color: #27b1ff;
+      color: #0086d4;
+    }
+  }
+
+  &__mono ::ng-deep input {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)